source/expressions.tex

```diff
@@ -5378,7 +5378,7 @@
 and the deallocation function's name is
 \tcode{\keyword{operator} \keyword{delete}[]}.
 \begin{note}
-An implementation is required to provide default definitions for the global
+An implementation is expected to provide default definitions for the global
```

Comment:
```suggestion
An implementation provides default definitions for the global
```

Don't we require it, not just expect it to be true for most implementations?

Reply:
Well spotted. Can we make this a separate issue?

Reply:
This is now #6761